src/python-common/ceph/deployment/service_spec.py

```diff
@@ -502,6 +502,8 @@ def __init__(self,
                  rgw_zone=None,  # type: Optional[str]
                  subcluster=None,  # type: Optional[str]
                  rgw_frontend_port=None,  # type: Optional[int]
+                 rgw_frontend_ssl_certificate=None, # type Optional[List[str]]
+                 rgw_frontend_ssl_key=None, # type: Optional[List[str]]
```

Comment:
ops, forgot to put these in variables for RGWSpec. Will be fixed in the next version

Reply:
`make check` is currently failing on this ...